### ISSUE TITLE
In Scene & Resources Tabs selecting node does not display node in 2D or Script Tab consistently

### ISSUE THREAD

**Issue by RefinedSoftwareLLC** [godotengine/godot#662]
In Scene & Resources Tabs selecting node does not display node in 2D or Script Tab consistently.

Create a scene, a node it in, link a script to that node, create a few subnodes, create another script not linked to any nodes.

Correct way: When you are in viewing the 2D Tab, select the Resources Tab and select a script that is NOT currently selected then it automatically changes to the Script Tab with that script open.
Bug: When you are in viewing the 2D Tab, select the Resources Tab and select the script that IS currently selected then it does NOT automatically change to the Script Tab with that script open.

Correct way: When you are in viewing the Script Tab, select the Scene Tab and select a node that is NOT the root node with the script then it automatically changes to the 2D Tab with that node open.
Bug: When you are in viewing the Script Tab, select the Scene Tab and select a node that IS the root node with the script then it does NOT automatically change to the 2D Tab with that node open.

**Comment by akien-mga** [comment]
I'm sorry, I tried to understand the issue to reproduce it but the procedure you describe is kind of hard to follow... Many things changed since Aug 2014 in the way scenes are displayed, and the resources tab does not exist anymore, so I'll assume this bug is no longer valid.
